app/listings/page.tsx

```
@@ -277,7 +293,7 @@ export default function ListingsPage() {
                       .filter(
                         (sub) =>
                           sub.parent_category_id ===
-                          Number(selectedCategories[0]),
+                          parseInt(selectedCategories[0], 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)